src/redux/store/store.js

```diff
@@ -0,0 +1,10 @@
+import { configureStore } from "@reduxjs/toolkit";
+import loginSlice from "../reducers/loginSlice";
+import thunk from "redux-thunk";
+
+const store = configureStore({
+  reducer: loginSlice,
+  middleware: (applyMiddleware) => applyMiddleware(thunk),
+});
+
```

Comment:
you can export dispatch like this
export const { dispatch } = store
and you need call this dispatch inside any component , 
you don't need to import useDispatch hook